### PR TITLE
Add Crashes functional tests

### PR DESCRIPTION
**Pull request by Jamminroot**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [ ] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the Windows code?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

This PR adds Crashes functional tests to test project.

## Related PRs or issues

[AB#74720](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/74720)

### CODE REVIEW

File: SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Android/Crashes.cs [hunk 1]
```diff
@@ -295,5 +295,9 @@ public bool ShouldProcess(AndroidErrorReport androidReport)
                 return ShouldProcessErrorReport(report);
             }
         }
+        internal static void PlatformUnsetInstance()
```
Comment (guperrot) [inline SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Android/Crashes.cs]:
```suggestion

        internal static void PlatformUnsetInstance()
```

File: Tests/Microsoft.AppCenter.Test.Functional/Analytics/TrackEventTest.cs [hunk 2]
```diff
@@ -18,6 +18,7 @@ public class TrackEventTest
         [Fact]
         public async Task TrackEventWithoutPropertiesAsync()
         {
+
```
Comment (guperrot) [inline Tests/Microsoft.AppCenter.Test.Functional/Analytics/TrackEventTest.cs]:
```suggestion
```

File: Tests/Microsoft.AppCenter.Test.Functional/Analytics/TrackEventTest.cs [hunk 3]
```diff
@@ -49,6 +50,7 @@ public async Task TrackEventWithoutPropertiesAsync()
         [Fact]
         public async Task TrackEventWithPropertiesAsync()
         {
+
```
Comment (guperrot) [inline Tests/Microsoft.AppCenter.Test.Functional/Analytics/TrackEventTest.cs]:
```suggestion
```

**Comment by MatkovIvan** [comment]
Merging to feature branch to combine fixes across modules

**Pull request merged**